app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -8,18 +8,16 @@ $govuk-cookie-banner-background: govuk-colour("light-grey", "grey-4");
 }
 
 .gem-c-cookie-banner {
-  @include govuk-font($size: 16);
-  padding: govuk-spacing(2) 0;
   background-color: $govuk-cookie-banner-background;
 }
 
-// Only show accept button if users have js and can accept
-.gem-c-cookie-banner__button-accept {
+// Only show buttons if users have js and can accept
+.gem-c-button.js-visible {
   display: none;
 }
 
-.js-enabled .gem-c-cookie-banner__button-accept {
-  display: inline-block;
+.js-enabled .gem-c-button.js-visible {
```

Comment:
This seems to be applying styling using a `js-` prefixed class, which is specifically for JS hooks only and not to be used for styling. Is there an alternative way this could be implemented? Could it be shown/hidden in the JS as required?

Reply:
> This seems to be applying styling using a `js-` prefixed class, which is specifically for JS hooks only and not to be used for styling. Is there an alternative way this could be implemented? Could it be shown/hidden in the JS as required?

I felt this was a reasonable way to do it as it is using js to apply the js-enabled class that this relies on. I can do this in the js code but that seemed like more overhead. I'll look into it.

Reply:
Fixed.